File: ClassNinteen/app/src/main/java/com/amaderhub/classninteen/MainActivity.java
package com.amaderhub.classninteen;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    TextView Tview;
    int count;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Tview = findViewById(R.id.tview);

        for (count= 1 ; count<= 10000 ; count++) {

            Tview.append(""+count);
        }

        //End Of Loop
    }
}

File: CarRaicingGame/app/src/main/java/com/thatsnajmul/carraicinggame/MainActivity.kt
package com.thatsnajmul.carraicinggame

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.LinearLayout
import android.widget.TextView

class MainActivity : AppCompatActivity(),GameTask {
    lateinit var rootLaout :LinearLayout
    lateinit var startBtn :Button
    lateinit var againStartButton :Button
    lateinit var mGameView: GameView
    lateinit var score: TextView
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        startBtn = findViewById(R.id.startBtn)
        againStartButton = findViewById(R.id.AgainStartBtn)
        rootLaout = findViewById(R.id.rootLayout)
        score = findViewById(R.id.score)
        mGameView = GameView(this, this)

        startBtn.setOnClickListener{
            mGameView.setBackgroundResource(R.drawable.roads)
            rootLaout.addView(mGameView)
            startBtn.visibility = View.GONE
            againStartButton.visibility = View.GONE
            score.visibility = View.GONE

        }
        againStartButton.setOnClickListener{
            mGameView.setBackgroundResource(R.drawable.roads)
            rootLaout.addView(mGameView)
            startBtn.visibility = View.GONE
            againStartButton.visibility = View.GONE
            score.visibility = View.GONE

        }

    }

    override fun closeGame(mScore: Int) {
        score.text = "Score : $mScore"
        rootLaout.removeView(mGameView)
        startBtn.visibility = View.GONE
        againStartButton.visibility = View.VISIBLE
        score.visibility = View.VISIBLE
    }

}